src/Modifiers/AggregateFacilityUseProviderModifier.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * AggregateFacilityUseProviderModifier
+ * 
+ * Removes the provider attribute from the aggregateFacilityUse
+ * property of an OrderedItem's FacilityUse.
+ * 
+ * Used to modify the C1/C2/B order output if the provider has
+ * been added in during construction of Slot OrderedItems
+ * 
+ * This typically happens because the Orders feed requires the
+ * provider to be present and the C1/C2/B steps require it to
+ * be omitted. It depends how your renderer for FacilityUse has
+ * been setup
+ */
+
+namespace OpenActive\Modifiers;
+
+class AggregateFacilityUseProviderModifier
+{
+    public function __invoke($class, $key, $value)
+    {
+        if ($key == 'aggregateFacilityUse') {
+            if (isset($value['provider'])) {
```

Comment:
Might be worth combining these two `if`s into a single statement

Reply:
Resolved by: https://github.com/openactive/models-php/pull/109/commits/80ef47ad0647c104453ece3775426cde1ad5d5af